design/docs/cross-env-migration.md

```
@@ -0,0 +1,58 @@
+# Cross-Environment Migration (V3 → V4)
+
+## Status
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)